docs/index.md

```diff
@@ -73,15 +73,15 @@ currently supported through the UI.
 ## Config File
 
 To define your `fsspec` filesystems, you will need to list them in the Jupyter config folder,
-inside a file named `~/.jupyter/jupyter-fsspec.yaml`. Here's a sample file:
+inside a file named `~/.jupyter/jupyter-fsspec.yaml`. Here's a sample file where for the local
+filesystem instances, `file://Users/finnmertens` is the server root path:
```

Comment:
Normally file paths on macOS/Linux are `file:///Users/xx` with three /, since the 3rd one is for root. Should we standardize on that?

Reply:
Agreed. Without three "/", it would be relative to the cwd. We probably want to encourage including "file:" and absolute paths, except for the special case of mirroring the cwd, which is what the default file browser does.